resources/prosody-plugins/mod_filter_iq_rayo.lua

```diff
@@ -2,6 +2,7 @@ local st = require "util.stanza";
 
 local token_util = module:require "token/util".new(module);
 local room_jid_match_rewrite = module:require "util".room_jid_match_rewrite;
+local apply_features_restriction = module:require "util".apply_features_restriction;
```

Comment:
this is still the old one

Reply:
oh, guess you haven't pushed that change yet

Reply:
I think I messed it up, cause I force pushed. But I see it correct now